src/models/player.rs

```diff
@@ -197,6 +200,15 @@ impl<E: Env + 'static> UpdateWithCtx<E> for Player {
                 let watched_effects =
                     watched_update(&mut self.watched, &self.meta_item, &self.library_item);
 
+                let skip_gaps_effects = skip_gaps_update::<E>(
```

Comment:
This call doesn't make much sense to be honest as we depend on both the `VideoParams` and `SeriesInfo` being set/updated which for `VideoParams` only happens in a separate action.

Reply:
The probably it makes more sense to just set it to None?

Reply:
Good point. It should better be set to none.